### DISCUSSION TITLE
Welcome to Outlook Google Calendar Sync Discussions!

### DISCUSSION THREAD

**Discussion by phw198** [phw198/OutlookGoogleCalendarSync#1882] [Announcements]
## 👋 Welcome!
  Discussions has been enabled to work out if it might make a useful place to connect with other members of the OGCS user community. The hope is it will allow you to:
  * Engage more easily than raising GitHub issues
  * Discuss issues more naturally in greater detail
  * Ask questions about how OGCS works, in a more informal manner
  * Share ideas and knowledge between like-minded users
  * Engage with other community members

So, to help things get started, drop a comment below on if you think Discussions will be helpful...

**Comment by heshie75v** [comment]
I have been using Outlook Google calendar sync for a while now, I cannot get it for some reason to work with my Outlook IOS app. Am I doing something wrong? .
Otherwise works great with my Google, and regular Outlook calendars .
Thanks in advance.

**Reply by phw198** [reply]
I've moved your question to a discussion thread of it's own on #1892 .

**Comment by cgitwom** [comment]
Google calendar events auto-created from gmail are not syncing to Outlook. I have tried changing the Google event visibility, but am not sure what else to try. Any suggestions?

**Reply by phw198** [reply]
https://github.com/phw198/OutlookGoogleCalendarSync/issues/1919#issuecomment-2395395518

**Comment by phw198** [comment]
The only comments made on this discussion thread so far are nothing to do with whether they are helpful 😆 

Though it is pretty indicative - so far, I'm mostly just finding Conversations cause confusion and Issues are better. Also, there is a general failure to use the threading mechanism in Discussions anyway, which is a large part of why I wanted to try them. But I'll leave the experiment going a bit longer...

**Comment by jflanner** [comment]
I've been using Calendar sync for a while.  It is a great tool.  Is there a trick to Category matching?  Everything in google shows up as the default color - blueberry.  On outlook itself, I have 5 categories set up each with a different color.  When I open the calendar on outlook - I immediately know what kind of day it will be based on the color if the items.  It would be fantastic if I can get these (or similar) colors on google.  

In settings - sync options - for items created in google calendar -  I have set color/category to Calendar default (blueberry) and in the what section I have colors/categories checked, and for mappings I have my categories with the Google color I want.  It does not seem to take.  Everything is still blueberry in Google.

Any help appreciated.

**Reply by phw198** [reply]
The "for items created in Google" set to blueberry is an _override_ - it will set everything OGCS syncs to Google as blueberry which is what you are seeing.

If you want the categories that are in Outlook to sync to Google colours, unset that and just rely on the What section for colours/categories (setting  mappings if you want it to use specific colours etc). 

You'll need to do a full sync from O->G to update already synced items - press `Shift` and click `Full Sync`.

**Reply by jflanner** [reply]
AWESOME Paul.  Thank You!!!
